packages/yew/src/functional/mod.rs

```diff
@@ -23,6 +23,7 @@ use std::rc::Rc;
 mod hooks;
 pub use hooks::*;
 
+pub use self::function_component as fc;
```

Comment:
Not a fan of this alias. Users can always alias it themselves if they really want to shorten it, but I don't think we should encourage it because of the reduced clarity and because it adds multiple ways to do the same thing.

I wouldn't mind shortening `function_component` to just `component` in the future, since the attribute placement already makes it clear that it's obviously a function and even if we were to support the attribute on something like a struct, we could always detect that within the macro.

Reply:
@siku2 We kind of copy pasted the idea of having `FC` shorthand from react :D
But yeah when given some thought `component` does sound better and is not so long to make you want to alias it.

Reply:
I agree here. I removed the alias.